src/components/Header.jsx

```diff
@@ -0,0 +1,145 @@
+        <stop id='stop3712' stopColor='#f5c8c7' offset='.5' />
+        <stop id='stop3716' stopColor='#cf8885' offset='.75' />
+        <stop id='stop3694' stopColor='#aa4944' offset='1' />
+      </linearGradient>
+      <linearGradient id='linearGradient3724' y2='549.69' gradientUnits='userSpaceOnUse' x2='247.02' y1='538.62' x1='235.89'>
+        <stop id='stop3720' stopColor='#ddb090' offset='0' />
+        <stop id='stop3726' stopColor='#ebd1bf' offset='.5' />
+        <stop id='stop3722' stopColor='#ce8d60' offset='1' />
+      </linearGradient>
+    </defs>
+    <g id='layer1' transform='translate(-229.26 -482.76)'>
+      <path id='path2830' d='m229.76 555.4 3.2561-9.2703 6.923-19.71s42.261-41.677 44.49-42.78c2.2293-1.1029 4.573 0.56973 4.573 0.56973s4.6354 2.0837 8.8807 6.9637c4.2452 4.88 6.3782 7.5724 6.23 10.106-0.1483 2.5336-44.48 43.032-44.48 43.032l-19.735 7.3258-10.138 3.7632z' stroke='#000' strokeWidth='1px' fill='url(#linearGradient3696)' />
+      <path id='path2834' d='m238.94 529.62 0.38756 3.2698 7.1132-2.1889 1.3838 9.042 7.9794 0.42929 0.7179 5.1518 3.1176-1.0132 18.5-17.192s-3.9188-6.1573-9.3203-11.495c-5.3588-5.2956-9.8054-7.9366-9.8054-7.9366l-19.069 18.732-1.0054 3.2009z' stroke='#000' strokeWidth='1px' fill='#fba635' />
+      <path id='path2836' d='m246.44 530.7 19.09-18.227s1.1615 0.58399 4.7343 4.4807 4.7759 5.49 4.7759 5.49l-19.237 17.728-7.9794-0.42929-1.3838-9.042z' fill='#f5b25f' />
+      <path id='path2838' fill='#b54d0e' d='m256.52 545.32 3.1176-1.0132 18.5-17.192-3.0994-4.6745-19.237 17.728 0.7179 5.1518z' />
+      <path id='path2832' d='m233.35 545.72 6.2295 5.8246-9.8159 3.8538 3.5863-9.6784z' />
+      <path id='path2840' d='m238.94 529.62 0.38756 3.2698 7.1132-2.1889 1.3838 9.042 7.9794 0.42929 0.7179 5.1518 3.1176-1.0132 18.5-17.192s-3.9188-6.1573-9.3203-11.495c-5.3588-5.2956-9.8054-7.9366-9.8054-7.9366l-19.069 18.732-1.0054 3.2009z' stroke='#000' strokeWidth='1px' fill='none' />
+      <path id='path2842' d='m233.35 545.72 6.2295 5.8246 16.94-6.2219-0.7179-5.1518-7.9794-0.42929-1.3838-9.042-7.1132 2.1889-0.38756-3.2698-5.5873 16.101z' stroke='#000' strokeWidth='1px' fill='url(#linearGradient3724)' />
+      <path id='path2844' fill='#7e5f56' d='m273.13 494.59s6.4173 2.3893 10.652 7.017c4.2348 4.6277 8.9574 11.889 8.9574 11.889l-14.604 13.619s-6.3353-9.5962-9.4724-12.121c-3.1371-2.5246-9.6533-7.311-9.6533-7.311l14.12-13.093z' />
+      <path id='path2846' fill='#d6c8c6' d='m259.01 507.69 6.5161 4.7864 14.588-13.997-6.9844-3.8825-14.12 13.093' />
+      <path id='path2848' d='m265.53 512.47 3.8736 3.2765 5.6365 6.6943 14.372-13.104s-1.6856-4.1009-3.6664-6.1677c-1.9808-2.0667-5.6278-4.6966-5.6278-4.6966' fill='#ece4e2' />
+      <path id='path2850' d='m273.13 494.59s6.4173 2.3893 10.652 7.017c4.2348 4.6277 8.9574 11.889 8.9574 11.889l-14.604 13.619s-5.7015-8.2713-8.7359-11.369c-2.8179-2.8765-10.39-8.0629-10.39-8.0629l14.12-13.093z' stroke='#000' strokeWidth='1px' fill='none' />
+    </g>
+  </svg>
+
+)
```

Comment:
Maybe we can move this into an .svg file and import it? They do that with the CreateReactApp logo...